.firebase/hosting.ZGlzdA.cache

```diff
@@ -0,0 +1,86 @@
+index.html,1649211765973,62a5585f561550a0d198636336fee5ddbe2d547f119845d49f18e86d960b5fe4
+favicon.ico,1649211765973,1e71457865f706dc865b49a54a86e193818220d290b30226b6630a42faf1535d
+css/app.86864ba3.css,1649211765980,239df9de8ae5308d7485e7b517c84dd39f13d9707ff81f9cfc670cc133fc87f8
+images/avatars/avatar0.jpg,1649211765982,291c3b50763bd49190abd30b020258e0b163bb7be798d5f3f6edaa0e3a86ad4c
+images/avatars/avatar1.jpg,1649211765982,2ac306301432d95caddd9b3821d6f920374dbdcaa2880aeefae54c0500e11086
+images/avatars/avatar10.jpg,1649211765982,32e7c3bd86c97403e0d73ff83bb0df0db18364f6f05cc6669dd228f97b11c249
+images/avatars/avatar11.jpg,1649211765982,373769ff6a5ab2f974392a343b8afa0984fba82c7a3641406b716fd830b16437
+images/avatars/avatar2.jpg,1649211765982,8910546809388868c2550fac285e8c491c4f4cabffecbb67019eadf833e66edb
+images/avatars/avatar3.jpg,1649211765982,3116058658219f143442d7fdb501b8386463e669481f199d39623c60b5f567c2
```

Comment:
You should ignore these files. These will be updated every time you deploy, no matter if there are any actual changes in the code or not.